mozilla/blurts-server
# Do not reinit db connection locally

# Description

When developing locally we reinitialize and create a new database connection each time the server reloads. When hitting your local db client threshold this results in an `Unhandled Runtime Error` you have likely already had the pleasure to enjoy instead of your latest code changes.

For local development we can just store the initialized db in `global` and hopefully you’ll have to `npm run dev` a lot less now.

<img width="355" alt="image" src="https://github.com/mozilla/blurts-server/assets/13835474/463d878e-9ba7-4884-b3b1-415535eea367">

# How to test

Try developing and making changes for a bit on/from this branch. If you are not running into the `Error: sorry, too many clients already` view or any new issues related to your local database connection all should be good. :)
